diff --git a/node_modules/react-native/src/private/webapis/dom/events/Event.js b/node_modules/react-native/src/private/webapis/dom/events/Event.js
index f918f97..01a0e6b 100644
--- a/node_modules/react-native/src/private/webapis/dom/events/Event.js
+++ b/node_modules/react-native/src/private/webapis/dom/events/Event.js
@@ -193,6 +193,7 @@ export default class Event {
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event, 'NONE', {
+  writable: true,
   enumerable: true,
   value: 0,
 });
@@ -200,42 +201,49 @@ Object.defineProperty(Event, 'NONE', {
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event.prototype, 'NONE', {
   enumerable: true,
+  writable: true,
   value: 0,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event, 'CAPTURING_PHASE', {
   enumerable: true,
+  writable: true,
   value: 1,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event.prototype, 'CAPTURING_PHASE', {
   enumerable: true,
+  writable: true,
   value: 1,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event, 'AT_TARGET', {
   enumerable: true,
+  writable: true,
   value: 2,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event.prototype, 'AT_TARGET', {
   enumerable: true,
+  writable: true,
   value: 2,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event, 'BUBBLING_PHASE', {
   enumerable: true,
+  writable: true,
   value: 3,
 });
 
 // $FlowExpectedError[cannot-write]
 Object.defineProperty(Event.prototype, 'BUBBLING_PHASE', {
   enumerable: true,
+  writable: true,
   value: 3,
 });
 
